src/main/java/net/blay09/mods/waystones/client/WaystoneCreativeTab.java

```diff
@@ -0,0 +1,21 @@
+package net.blay09.mods.waystones.client;
+
+import net.blay09.mods.waystones.Waystones;
+import net.minecraft.item.ItemStack;
+
+public class WaystoneCreativeTab extends net.minecraft.creativetab.CreativeTabs {
```

Comment:
Thanks, I am used to not use anonymous classes, but you are right this is small enough to just write it in. Will do.